### DISCUSSION TITLE
Incorrect or missing translation for validation errors in Farsi language

### DISCUSSION THREAD

**Discussion by alihoseiny** [encode/django-rest-framework#9355] [Potential Issue] [answered]
Hello,

There are some incorrect translations in the Farsi translation file. For example, the phrase "Upload a valid image. The file you uploaded was either not an image or a corrupted image." has been translated as:
 "یک عکس معتبر آپلود کنید. فایلی که ارسال کردید عکس یا عکس خراب شده نیست". 
If you translate the end part into English, it means: "The image is not corrupted".

Additionally, there are missing translations and other typo errors.

Interestingly, the repository suggests that we should create a Discussion before sending fixes. So, what is the next step? Could I create a PR?

**Comment by TGoddessana** [comment, marked as answer]
Check this out 😃

https://www.django-rest-framework.org/community/project-management/#translations
